apps/etterlatte-behandling/src/main/kotlin/sak/SakDao.kt

```diff
@@ -31,11 +31,14 @@ class SakDao(private val connection: () -> Connection) {
         }
     }
 
-    fun opprettSak(fnr: String, type: SakType): Sak {
+    fun opprettSak(fnr: String, type: SakType, enhet: String? = null): Sak {
         val statement =
-            connection().prepareStatement("INSERT INTO sak(sakType, fnr) VALUES(?, ?) RETURNING id, sakType, fnr")
+            connection().prepareStatement(
+                "INSERT INTO sak(sakType, fnr, enhet) VALUES(?, ?, ?) RETURNING id, sakType, fnr"
+            )
         statement.setString(1, type.name)
         statement.setString(2, fnr)
+        statement.setString(3, enhet)
```

Comment:
Lurer på om denne kan lagre strengen "NULL" i stedet for raden `NULL` når `enhet` er `null`. Noe ala forslaget vil i så fall hjelpe (kjedelig å begge typer verdier i denne kolonnen)

```suggestion
        enhet?.let { statement.setString(3, it) } ?: statement.setNull(3) // setNull krever noe ekstra IIRC
```

Reply:
Litt idea og litt google peker på at det skal være:

`enhet?.let { statement.setString(3, it) } ?: statement.setNull(3, Types.VARCHAR)`